grammars/css.cson

```diff
@@ -20,294 +20,1176 @@
-          '2':
-            'name': 'punctuation.section.function.css'
-        'end': '(\\))'
+            'name': 'punctuation.definition.parameters.begin.bracket.round.css'
+        'end': '\\)'
         'endCaptures':
-          '1':
-            'name': 'punctuation.section.function.css'
+          '0':
+            'name': 'punctuation.definition.parameters.end.bracket.round.css'
         'patterns': [
           {
-            'match': '(?x)\\b\n\t\t\t\t\t\t\t    (0*((1?[0-9]{1,2})|(2([0-4][0-9]|5[0-5])))\\s*,\\s*){2}\n\t\t\t\t\t\t\t    (0*((1?[0-9]{1,2})|(2([0-4][0-9]|5[0-5])))\\b)\n\t\t\t\t\t\t\t    (\\s*,\\s*((0?\\.[0-9]+)|[0-1]))?\n\t\t\t\t\t\t\t'
-            'name': 'constant.other.color.rgb-value.css'
-          }
-          {
-            'match': '\\b([0-9]{1,2}|100)\\s*%,\\s*([0-9]{1,2}|100)\\s*%,\\s*([0-9]{1,2}|100)\\s*%'
-            'name': 'constant.other.color.rgb-percentage.css'
+            'include': '#media-features'
           }
           {
-            'include': '#numeric-values'
+            'include': '#media-feature-keywords'
           }
-        ]
-      }
-    ]
-  'comment-block':
-    'begin': '/\\*'
-    'captures':
-      '0':
-        'name': 'punctuation.definition.comment.css'
-    'end': '\\*/'
-    'name': 'comment.block.css'
-  'media-query':
-    'begin': '(?i)\\s*(only|not)?\\s*(all|aural|braille|embossed|handheld|print|projection|screen|tty|tv)?'
-    'beginCaptures':
-      '1':
-        'name': 'keyword.operator.logic.media.css'
-      '2':
-        'name': 'support.constant.media.css'
-    'end': '\\s*(?:(,)|(?=[{;]))'
-    'endCaptures':
-      '1':
-        'name': 'punctuation.definition.arbitrary-repetition.css'
-    'patterns': [
-      {
-        'include': '#comment-block'
-      }
-      {
-        'begin': '\\s*(and)?\\s*(\\()\\s*'
-        'beginCaptures':
-          '1':
-            'name': 'keyword.operator.logic.media.css'
-        'end': '\\)'
-        'patterns': [
           {
-            'begin': '(?x)\n\t                            (\n\t                                ((min|max)-)?\n\t                                (\n\t                                    ((device-)?(height|width|aspect-ratio))|\n\t                                    (color(-index)?)|monochrome|resolution\n\t                                )\n\t                            )|grid|scan|orientation\n\t                            \\s*(?=[:)])'
-            'beginCaptures':
-              '0':
-                'name': 'support.type.property-name.media.css'
-            'end': '(:)|(?=\\))'
-            'endCaptures':
-              '1':
-                'name': 'punctuation.separator.key-value.css'
+            'match': ':'
+            'name': 'punctuation.separator.key-value.css'
           }
           {
-            'match': '(?<![\\w-])(portrait|landscape|progressive|interlace)(?![\\w-])'
-            'name': 'support.constant.property-value.css'
+            'match': '[><]=|[=<>]'
```

Comment:
I find this harder to understand at a glance than if they were written out like before.